extensions/adobe/experience/decisioning/offerItem.schema.json

```diff
@@ -0,0 +1,123 @@
+{
+  "meta:license": [
+    "Copyright 2023 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/decisioning/offeritem",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Offer Item",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "meta:status": "experimental",
+  "description": "Class which represents an Offer",
+  "definitions": {
+    "offeritem": {
+      "properties": {
+        "xdm:lifecycleStatus": {
+          "title": "life cycle status",
+          "type": "string",
+          "description": "Status associated with offer life cycle.",
+          "meta:enum": {
+            "draft": "Draft",
+            "approved": "Approved",
+            "archived": "Archived"
+          }
+        },
+        "xdm:frequencyCappingConstraints": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "title": "Frequency capping constraint",
+            "description": "Frequency capping constraint define additional rules based on prior offer proposition, either across all audiences or for a particular profile identity.",
+            "properties": {
+              "xdm:limit": {
+                "title": "Limit",
+                "description": "How many times an option can be proposed.",
+                "type": "integer",
+                "minimum": 1
+              },
+              "xdm:scope": {
+                "title": "scope",
+                "description": "Scope of the capping",
+                "type": "string",
+                "enum": ["global", "profile"],
+                "meta:enum": {
+                  "global": "Global cap is a constraint on how many times an option can be proposed in totality.",
```

Comment:
These meta:enums are too long for drop down. Please shorten them to fit the UI experience .